content/structure.md

```diff
@@ -19,19 +19,19 @@ Meteor is a *full-stack* framework for building applications; this means Meteor
 
-Since this is a new feature introduced in Meteor 1.3, you will find a lot of code online that uses the older, more centralized conventions built around packages and apps declaring global symbols. This old system still works, so to opt-in to the new module system code must be placed in the `imports/` directory in your application. We expect a future release of Meteor will turn on modules by default for all code, because this is more aligned with how developers in the wider JavaScript community write their code.
+Since this is a new feature introduced in Meteor 1.3, you will find a lot of code online that uses the older, more centralized conventions built around packages and apps declaring global symbols. This old system still works, so to opt-in to the new module system code must be placed inside the `imports/` directory in your application. We expect a future release of Meteor will turn on modules by default for all code, because this is more aligned with how developers in the wider JavaScript community write their code.
 
 You can read about the module system in detail in the [`modules` package README](https://docs.meteor.com/#/full/modules). This package is automatically included in every new Meteor app as part of the `ecmascript` meta-package, so most apps won't need to do anything to start using modules right away.
 
 <h3 id="importing-from-packages">Importing from packages</h3>
 
-In Meteor, it is also simple and straightforward to use the `import` syntax to load npm packages on the client or server, and access the package's exported symbols as you would with any other module. You can also import from Atmosphere packages, but the import path must be prefixed with `meteor` to avoid conflict with the npm package namespace. For example, to import `HTTP` you can do `import { HTTP } from 'meteor/http'`.
+In Meteor, it is also simple and straightforward to use the `import` syntax to load npm packages on the client or server, and access the package's exported symbols as you would with any other module. You can also import from Atmosphere packages, but the import path must be prefixed with `meteor/` to avoid conflict with the npm package namespace. For example, to import `HTTP` you could use `import { HTTP } from 'meteor/http'`.
 
 <h3 id="using-require">Using `require`</h3>
 
-In Meteor, `import` statements compile to CommonJS `require` syntax. However, we encourage you to use `import`. 
+In Meteor, `import` statements compile to CommonJS `require` syntax. However, as a cenvention we encourage you to use `import`. 
```

Comment:
I've corrected the typo.
My thought was that there is no explanation whatsoever why "we encourage you to use `import`". Why do you encourage it? Well, at least because it's a useful convention.

Happy to remove this, if you feel it's a pointless insertion.